### ISSUE TITLE
throwOnImmutableMutation being hit when trying to send a message

### ISSUE THREAD

**Issue by poffdeluxe** [getsentry/react-native-sentry#57]
Happening on iOS. Running react-native 0.42.3 and react-native-sentry 0.8.3

My index.os.js looks like this:
```
import { AppRegistry } from 'react-native';
import App from './App';

import {
  Sentry,
  SentrySeverity,
  SentryLog
} from 'react-native-sentry';

Sentry.config("[MY DSN HERE]", {
  // deactivateStacktraceMerging: true,
  logLevel: SentryLog.Debug
}).install();

AppRegistry.registerComponent('governedapp', () => App);
```

In my code, I'm attempting to send a message with:
```
      Sentry.captureMessage("My message", {
        level: SentrySeverity.Error
      });
```

but I'm getting this crash:
<img width="413" alt="glass_and_iphone_7_ _ios_10_3__14e269_" src="https://cloud.githubusercontent.com/assets/766596/26059474/83acc44c-3947-11e7-892e-7cd4002fcbb4.png">
(I blurred out the key value since I think it's specific to the firebase integration I'm running)

It looks like this is happening when trying to prepare a list of breadcrumbs to send with my message. Seems like a non-release build specific error as well.

**Comment by poffdeluxe** [comment]
Was able to workaround this by turning off xhr breadcrumbs:
```
Sentry.config("[DSN]", {
  logLevel: SentryLog.Debug,
  autoBreadcrumbs: {
    'xhr': false,
    'console': true,
    'dom': true,
    'location': true 
  }
}).install();
```

**Comment by HazAT** [comment]
Fixed in `v0.12.10`
